stdpopsim/homo_sapiens.py

```diff
@@ -696,3 +696,95 @@ def __init__(self):
             msprime.MassMigration(
                 time=T_nean_split, source=3, destination=0, proportion=1.0)
         ]
+
+
+class SchiffelsZigzag(models.Model):
+    """
+    Model Name:
+        SchiffelsZigzag
+
+    Model Description:
+        A validation model used by Schiffels and Durbin (2014) and Terhorst and Terhorst, Kamm, and Song (2017) with periods
+        of exponential growth and decline in a single population.
+        The original SCRM command for this model was:
```

Comment:
What's the reasoning for including this in the documentation? I don't see how it's going to help users TBH; presumably this provenance is somewhere in the original papers if we need to track it down?

Reply:
You're right, given that it's probably best to just natively code the model this is superfluous. I've removed it.